pandas/tests/tools/test_to_datetime.py

```diff
@@ -1066,6 +1066,25 @@ def test_convert_object_to_datetime_with_cache(
         )
         tm.assert_series_equal(result_series, expected_series)
 
+    @pytest.mark.parametrize("cache", [True, False])
+    @pytest.mark.parametrize(
+        ("input", "expected"),
+        (
+            (
+                Series([NaT] * 200 + [None] * 200, dtype="object"),
```

Comment:
Mind using less values here? (20/40 should be good)

Reply:
don't we have a predefined breakpoitn with 50 values?

Reply:
Yeah, but IIRC that's needed when specifically testing caching behavior